reflex/utils/hosting.py

```diff
@@ -448,14 +449,18 @@ def deploy(
             )
         # If the server explicitly states bad request,
         # display a different error
-        if response.status_code == HTTPStatus.BAD_REQUEST:
-            raise ValueError(response.json()["detail"])
+        assert (
```

Comment:
Nit: Use of assert in production code is generally not recommended.  https://snyk.io/blog/the-dangers-of-assert-in-python/

Reply:
Ohhhh, I did not know that. I've replaced all my assertions with manual if/elses.